spotinst/commons/common_organization_policy.go

```diff
@@ -0,0 +1,117 @@
+package commons
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/organization"
+	"log"
+)
+
+const (
+	OrgPolicyResourceName ResourceName = "spotinst_organization_policy"
+)
+
+var OrgPolicyResource *OrgPolicyTerraformResource
+
+type OrgPolicyTerraformResource struct {
+	GenericResource
+}
+
+type OrgPolicyWrapper struct {
+	OrgPolicy *organization.Policy
+}
+
+func NewOrgPolicyResource(fieldsMap map[FieldName]*GenericField) *OrgPolicyTerraformResource {
+	return &OrgPolicyTerraformResource{
+		GenericResource: GenericResource{
+			resourceName: OrgPolicyResourceName,
+			fields:       NewGenericFields(fieldsMap),
+		},
+	}
+}
+
```

Comment:
let's the hierarchy same across all resources: OnCreate, OnRead, OnUpdate

Reply:
changed in same order